src/server.js

```diff
@@ -3,10 +3,20 @@ const server = require('@dvcorg/websites-server')
 
 const app = server.app
 
+const helmetOptions = {
+  contentSecurityPolicy: {
+    directives: {
+      frameSrc: ['https://embed.testimonial.to']
+    }
+  }
+}
```

Comment:
We'll have to remember to move this when we do https://github.com/iterative/dvc.org/issues/3898